### PR TITLE
SSCS-10075 Implement PUT partiesNotified Endpoint

### PR DESCRIPTION
**Pull request by TeaWithLucas**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SSCS-10075

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[x] Yes
[ ] No
```

### CODE REVIEW

**Review by ikirsanov — COMMENTED** [review]
Could you please rollback changes in OverridesMapping.class on 77-78 lines
`log.info("Default Override Fields set to {} for Case {}",
            defaultOverrideFields, wrapper.getCaseData());`

it should be 
`log.debug("Default Override Fields set to {} for Case ID {}",
            defaultOverrideFields,
            wrapper.getCaseData().getCcdCaseId());`

File: src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/PartiesNotifiedMapping.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.sscs.helper.mapping;
+
+import uk.gov.hmcts.reform.sscs.model.HearingWrapper;
+import uk.gov.hmcts.reform.sscs.model.partiesnotified.PartiesNotifiedRequestPayload;
+import uk.gov.hmcts.reform.sscs.model.partiesnotified.ServiceData;
+
+import static uk.gov.hmcts.reform.sscs.helper.service.HearingsServiceHelper.getVersion;
+
+public final class PartiesNotifiedMapping {
+    private PartiesNotifiedMapping() {
```
Comment (ikirsanov) [inline src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/PartiesNotifiedMapping.java:10]: @NoArgsConstructor(access = AccessLevel.PRIVATE) just a suggestion.

Comment (TeaWithLucas) [reply src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/PartiesNotifiedMapping.java:10]: Doesn't work unfortuantely, tried it before, pmd will flag it as a helper without a private constructor :(

Comment (ikirsanov) [reply src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/PartiesNotifiedMapping.java:10]: Strange, alright no worries

**Review by ikirsanov — APPROVED** [review]

**Pull request closed without merging**